### PR TITLE
DX-116 add intercept force404 example recipe

### PR DESCRIPTION
**Pull request by mccataldo**
Closes DX-116

### CODE REVIEW

**Comment by mccataldo** [comment]
@cowboy @amirrustam any other reviewers required on PRs for this repo?

File: examples/stubbing-spying__intercept/cypress/integration/intercept-force404.spec.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/// <reference types="Cypress" />
+
+describe("intercept", () => {
```
Comment (cowboy) [inline examples/stubbing-spying__intercept/cypress/integration/intercept-force404.spec.js:3]: The `cy.route` `force404` option docs say this:
> Forcibly send a 404 status when the XHR does not match any existing `cy.route()`.

In order for this example to cover the `force404` use case, it should show how to define at least one explicit intercept along with the 404 "default" and then show, in a test, how the defined network request will pass, but an undefined request will error.

**Review by cowboy — CHANGES_REQUESTED** [review]

File: examples/stubbing-spying__intercept/cypress/integration/intercept-force404.spec.js (hunk 1 above)
Comment (mccataldo) [reply examples/stubbing-spying__intercept/cypress/integration/intercept-force404.spec.js]: Done

**Review by cowboy — APPROVED** [review]

File: examples/stubbing-spying__intercept/cypress/integration/intercept-force404.spec.js (hunk 1 above)
Comment (mccataldo) [reply examples/stubbing-spying__intercept/cypress/integration/intercept-force404.spec.js]: @cowboy if there's nothing else on this one, please resolve it.

**Review by cowboy — APPROVED** [review]

**Review by cowboy — APPROVED** [review]

**Pull request merged**